src/app/Http/ViewHelpers/Employee/EmployeeLogViewHelper.php

```diff
@@ -49,4 +50,20 @@ public static function employee(Employee $employee): array
             'avatar' => ImageHelper::getAvatar($employee, 80),
         ];
     }
+
+    /**
+     * Log Types for the employee.
+     */
+    public static function employeeLogTypes(Employee $employee): array
+    {
+        $logTypes = DB::table('employee_logs')
```

Comment:
Could you refactor this to use Eloquent instead of the Query Builder? We tend to lean towards using Eloquent as much as possible and only the Query Builder if absolutely necessary.

Reply:
This has been refactored to use Eloquent.